### ISSUE TITLE
example for Creating models with the Core API

### ISSUE THREAD

**Issue by aihuawu** [tensorflow/tfjs#1562]
https://tensorflow.google.cn/js/guide/models_and_layers

1)  The layer functional model
const input = tf.input({shape: [784]});
const dense1 = tf.layers.dense({units: 32, activation: 'relu'}).apply(input);
const dense2 = tf.layers.dense({units: 10, activation: 'softmax'}).apply(dense1);
const model = tf.model({inputs: input, outputs: dense2});

2) Core API
const w1 = tf.variable(tf.randomNormal([784, 32]));
const b1 = tf.variable(tf.randomNormal([32]));
const w2 = tf.variable(tf.randomNormal([32, 10]));
const b2 = tf.variable(tf.randomNormal([10]));

function model(x) {
  return x.matMul(w1).add(b1).relu().matMul(w2).add(b2).softmax();
}

What's x here? what is returned?
Is x a tensor or SymbolicTensor?
Is the returned value a tensor, SymbolicTensor, or Model?

I want it to return a model so that I can compile and fit with tensor or dataset.
The document is not clear how to use core api to build model.

george wu

**Comment by bileschi** [comment]
Hello, this sort of question is better suited to StackOverflow.  This issues list is for bugs and feature requests.  Thanks and please use the tag 'tensorflow.js' on your StackOverflow question to makes sure it's discoverable.
